hikari/impl/rest.py

```diff
@@ -1481,8 +1481,8 @@ async def delete_messages(
             # (albeit at a cost of maybe a couple-dozen milliseconds per call),
             # I am just gonna invoke these sequentially instead.
             try:
-                if len(pending) == 1:
-                    message = pending[0]
+                if len(chunk) == 1:
+                    message = chunk[0]
                     try:
                         await self.delete_message(channel, message)
                     except errors.NotFoundError as exc:
```

Comment:
Not a specific problem with this PR, just wondering why this uses except as `exc` vs `ex` below...

Reply:
Saw this in other files too but whatever.

Reply:
Was already there when I got to it. I'd personally use `e` but oh well.

Reply:
Since 39767a9 should this be changed across the whole lib? Preferably in another PR obviously.

Reply:
does it even matter that much? It's just a small detail

Reply:
True but now there's 3 conventions for errors lol

Reply:
L